src/adapters/task/mapper/task.mapper.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable functional/no-let */
+import { TaskBuilder } from 'domain/task/builder/task/task.builder'
+import type { Task, UpdateTask } from 'domain/task/entity/task'
+import type { AmendTask } from 'domain/task/command/amendtask'
+import type { CreateTask } from 'domain/task/command/createTask'
+import type { DeepReadonly } from 'superTypes'
+import { UpdateTaskBuilder } from 'domain/task/builder/updateTask/updateTask.builder'
+
+export class TaskMapper {
+  public static readonly mapCreateTaskToEntity = (payload: DeepReadonly<CreateTask>): Task => {
```

Comment:
IMHO, there should be no mapping between the command and the entity, since it is the role of the CPU to manage it. So I would advise to delete it.

Reply:
OK, the builder is directly called in the reducer itself?

Reply:
Not if the builder has business logic, because if it is the case, it causes a problem of responsibility.